diff --git a/node_modules/@makerdao/dai-ui-icons/lib/index.js b/node_modules/@makerdao/dai-ui-icons/lib/index.js
index 7f597f9..a8a4597 100644
--- a/node_modules/@makerdao/dai-ui-icons/lib/index.js
+++ b/node_modules/@makerdao/dai-ui-icons/lib/index.js
@@ -504,12 +504,20 @@ export const icons = {
   },
   home: {
     path: (
-      <path
-        fillRule="evenodd"
-        clipRule="evenodd"
-        d="M18 7.95543L9.99993 1.73315L2 7.95532V10.489L9.99993 4.26688L18 10.4892V7.95543ZM3 16V10.9783L5 9.42276V16H7V13C7 12.4478 7.44772 12 8 12H12C12.5523 12 13 12.4478 13 13V16H15V9.42261L17 10.9782V16C17 17.1046 16.1046 18 15 18H12C11.4477 18 11 17.5523 11 17V14H9V17C9 17.5523 8.55228 18 8 18H5C3.89543 18 3 17.1046 3 16Z"
-        fill="currentColor"
-      />
+      <g>
+        <linearGradient id="Naamloos_verloop_28" x1="-1.44" y1="166.11" x2="97.1" y2="61.87"
+                        gradientUnits="userSpaceOnUse">
+          <stop offset="0" stop-color="#8484fb"/>
+          <stop offset="0.31" stop-color="#706ff5"/>
+          <stop offset="0.73" stop-color="#5b5aef"/>
+          <stop offset="1" stop-color="#5352ed"/>
+        </linearGradient>
+        <circle cx="80.49" cy="79.44" r="71.72" fill="url(#Naamloos_verloop_28)"/>
+        <circle cx="80.49" cy="79.44" r="65.72" fill="url(#Naamloos_verloop_28)" stroke="#fff" stroke-width="0.5px" stroke-miterlimit="10"/>
+        <circle cx="80.49" cy="79.44" r="65.27" fill="url(#Naamloos_verloop_28)" stroke="#fff" stroke-width="0.5px" stroke-miterlimit="10"/>
+        <path fill="#fff"
+              d="M62.77,50.78H46.61a1.54,1.54,0,0,0-1.42,2.15l5.32,12.45a1.55,1.55,0,0,0,1.42.93H68a1.55,1.55,0,0,0,1.43-2.14L64.19,51.73A1.53,1.53,0,0,0,62.77,50.78Zm35.66.94L82.57,89a1.54,1.54,0,0,1-2.85,0L75.53,78.93a1.54,1.54,0,0,0-1.43-1H58.23a1.54,1.54,0,0,0-1.42,2.15l14.34,33.58a1.56,1.56,0,0,0,1.43.94H88.33a1.55,1.55,0,0,0,1.42-.94L115.8,52.93a1.54,1.54,0,0,0-1.42-2.15H99.86A1.56,1.56,0,0,0,98.43,51.72ZM71,70.6h-16a1.54,1.54,0,0,0-1.42,2.15h0a1.54,1.54,0,0,0,1.42.94H71a1.55,1.55,0,0,0,1.43-2.14h0A1.56,1.56,0,0,0,71,70.6Z"/>
+      </g>
     ),
     viewBox: "0 0 20 20",
   },
